readme.txt

```diff
@@ -206,7 +204,7 @@ The AddressFinder website has a large [FAQ section](https://addressfinder.com.au
 = 1.2.12 =
 * Fixes errors when checkouts are missing address_line_2 field
 = 1.2.11 =
-* Checks window.AddressFinder is loaded before intialising plugin
+* Checks window.Addressfinder is loaded before intialising plugin
```

Comment:
If this is referencing what the actual source is doing, then I am not certain the source has changed the case from `AddressFinder` to `Addressfinder` 🤷